docs/_writing-tests/testharness-api.md

```diff
@@ -250,6 +250,30 @@ created Promise waiting to be fulfilled, or if the event is of a different type
 to the type currently expected. This ensures that only the events that are
 expected occur, in the correct order, and with the correct timing.
 
+If you cannot or do not want to use `EventWatcher` for a test case,
+`promise_test` allows you to mix traditional callback handling and promises by
+exposing the `done_promise` on the `Test` instance.
+
+An example:
+
+```js
+function foo() {
+  return Promise.resolve("foo");
+}
+
+promise_test(function(t) {
+  someObject.addEventListener("someevent", t.step_func(function() {
+    t.done();
+  }));
```

Comment:
When would this be called?

Reply:
I don't understand. (Maybe you can ping me in #whatwg later)

Reply:
I mean I don't understand this example. I don't see anything dispatching `someevent`.

Reply:
That's because nothing does. :slightly_smiling_face: It could be added.